build.gradle

```diff
@@ -1,20 +1,19 @@
 plugins {
-    id 'org.jetbrains.kotlin.jvm' version '1.3.72'
-    id 'com.gradle.plugin-publish' version '0.11.0'
+    id 'org.jetbrains.kotlin.jvm' version '1.4.32'
+    id 'com.gradle.plugin-publish' version '0.14.0'
     id 'jacoco'
     id 'java-gradle-plugin'
-    id 'pl.droidsonroids.jacoco.testkit' version '1.0.6'
+    id 'pl.droidsonroids.jacoco.testkit' version '1.0.7'
 }
 
 dependencies {
     compileOnly gradleApi()
-    implementation 'org.jetbrains.kotlin:kotlin-stdlib:1.3.72'
```

Comment:
stdlib dependency is now added automatically https://kotlinlang.org/docs/whatsnew14.html#dependency-on-the-standard-library-added-by-default